resources/Makefile.erb

```diff
@@ -1,11 +1,18 @@
 <% dirnames = get_directories.map {|d| d.path } %>
-export SHELL := $(shell which bash)
 
-tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
 workdir := $(PWD)
 
+export SHELL := "${workdir}/profiling_shell.sh"
+export VANAGON_STATSD_HOST ?= 10.32.162.156
+export VANAGON_STATSD_PORT ?= 8125
+export VANAGON_PROJECT ?= <%= @name %>
+export VANAGON_PLATFORM ?= <%= @platform.name.tr('-','.') %>
+
+tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
+
 all: file-list-before-build <%= package_name %>
 
+<%= package_name %>: export VANAGON_TARGET = generate-package
```

Comment:
This is the only additional piece of info I thought might be useful to gather beyond the <project>-build, <project>-configure, etc. make targets